mpa/det/inferrer.py

```diff
@@ -190,7 +186,24 @@ def dummy_dump_saliency_hook(model, input, out):
 
         metric = None
         if eval:
-            metric = dataset.evaluate(eval_predictions, **cfg.evaluation)[cfg.evaluation.metric]
+            metric = dataset.evaluate(eval_predictions, **cfg.evaluation)
+            metric = metric['mAP'] if isinstance(cfg.evaluation.metric, list) else metric[cfg.evaluation.metric]
+
+        # Check and unwrap ImageTilingDataset object from TaskAdaptEvalDataset
+        while hasattr(dataset, 'dataset') and not isinstance(dataset, ImageTilingDataset):
+            dataset = dataset.dataset
+
+        if isinstance(dataset, ImageTilingDataset):
```

Comment:
Supporting `ImageTilingDataset` on the way.. It is better to do it separately, but not sure that we can afford it in terms of time. I would approve to keep it here :)